models/users.model.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../db/connection.js");
+
+async function checkUserExists(username) {
```

Comment:
This function wouldn't be needed.
If a you send a comment with a username which isn't in the database PSQL will throw an error.